### PR TITLE
i18n(zh-cn): Add 6 error translations

### PR DESCRIPTION
**Pull request by XindiShang**
#### What kind of changes does this PR include?

- Translated content

#### Description

Added 6 Astro error translations:

`get-static-paths-expected-params.mdx`
`get-static-paths-invalid-route-param.mdx`
`get-static-paths-removed-rsshelper.mdx`
`invalid-get-static-path-param.mdx`
`invalid-get-static-paths-return.mdx`
`no-client-only-hint.mdx`

### CODE REVIEW

File: src/content/docs/zh-cn/reference/errors/get-static-paths-removed-rsshelper.mdx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+<DontEditWarning />
+
+
+> **GetStaticPathsRemovedRSSHelper**: RSS助手已从 `getStaticPaths` 中删除。请改用新的 @astrojs/rss 包。 (E03012)
+
+## 哪里出了问题？
+`getStaticPaths` 不再提供生成 RSS 摘要的助手函数。 我们建议迁移到 [@astrojs/rss](/zh-cn/guides/rss/#设置-astrojsrss) 集成。
+
+**请参阅：**
+-  [RSS Guide](/zh-cn/guides/rss/)
```
Comment (Yan-Thomas) [inline src/content/docs/zh-cn/reference/errors/get-static-paths-removed-rsshelper.mdx:17]: Pointing out this link might be untranslated, let me know if it's expected for "Guide" to not be translated here.

Comment (liruifengv) [reply src/content/docs/zh-cn/reference/errors/get-static-paths-removed-rsshelper.mdx:17]: I translated this https://github.com/withastro/docs/pull/2995

Comment (XindiShang) [reply src/content/docs/zh-cn/reference/errors/get-static-paths-removed-rsshelper.mdx]: @Yan-Thomas @liruifengv I changed the link translation to match the page title in the updated RSS guide, see #2995

**Review by liruifengv — APPROVED** [review]
Chinese LGTM.

**Pull request merged**